Gems/Atom/RHI/Vulkan/Code/Source/RHI/TransientAttachmentPool.cpp

```diff
@@ -144,7 +144,7 @@ namespace AZ
             if (descriptor.m_renderTargetBudgetInBytes || allowNoBudget)
             {
                 // Use an image descriptor of size 1x1 to get the memory requirements.
-                RHI::ImageBindFlags bindFlags = RHI::ImageBindFlags::Color | RHI::ImageBindFlags::DepthStencil | RHI::ImageBindFlags::ShaderRead;
```

Comment:
quick question - why was it depth stencil before?

Reply:
Save as comment above.

Reply:
So how is the support of Depth Stencil for this purpose on Vulkan for this purpose or is there no use case for that? (I also assume it is outside the scope of this change but want to verify)

Reply:
So Vulkan can create an image for depthstencil, or an image for color render target. But it can't create for both. 

With our pool implementation, we can create one pool which can create both type of images. 
The GetImageMemoryRequirements() function used here was only for get the memory alignment when create the pool.
I believe it works because creating an image as a color render target or an image as depth stencil have same memory alignment.